prow/deck/jobs/jobs.go

```diff
@@ -71,19 +71,28 @@ type serviceClusterClient interface {
 	ListProwJobs(selector string) ([]kube.ProwJob, error)
 }
 
-type podLogClient interface {
+type PodLogClient interface {
 	GetLog(pod string) ([]byte, error)
 }
 
-type configAgent interface {
+type ConfigAgent interface {
```

Comment:
something for another PR https://cs.k8s.io/?q=configAgent&i=nope&files=&repos=
this should be a common type, but we keep redefining it all over. it should probably be prow/config.Agent